app/views/estimate_flow/matter_type.html.slim

```diff
@@ -12,6 +12,9 @@
               label: { text: t(".immigration") }, hint: { text: t(".immigration_hint") }
       = form.govuk_radio_button :controlled_proceeding_type, MatterTypeForm::PROCEEDING_TYPES[:asylum],
               label: { text: t(".asylum") }, hint: { text: t(".asylum_hint") }
+      - if @estimate.level_of_help != "controlled"
```

Comment:
We'll also need to adjust the SubmitProceedingsService so that when this screen is shown and the domestic abuse case is not on the applicant form, we use the value from the MatterTypeForm (which should be renamed from `controlled_proceeding_type` to just `proceeding_type` I guess and not the value from ApplicantForm (which, to avoid duplicate keys in the session overwriting one another, should probably be renamed to something else)